_i18n/pl/keyboards.md

```diff
@@ -0,0 +1,1225 @@
+---
```

Comment:
Filip, the comments on this file are largely for my own benefit. I'll leave it up to you whether to update this file further.

----

Our infrastructure doesn't appear to have any capability to provide this page in any language other than English, because the file is generated every time the `qmk/qmk_firmware` repo is updated.